src/functions/uploadToRSIS/application/data-mapper/cat-cpc-mapper.ts

```diff
@@ -0,0 +1,111 @@
+import { ResultUpload } from '../result-client';
+import { DataField, FormType } from '../../domain/mi-export-data';
+import { TestData, TestResultCatCPCSchema } from '@dvsa/mes-test-schema/categories/CPC';
+import { addIfSet, field, mandatory, optional, optionalBoolean } from './data-mapper';
+import { formatApplicationReference } from '@dvsa/mes-microservice-common/domain/tars';
+import moment = require('moment');
+import { formatDateOfBirth, formatResult, formatLanguage } from './common-mapper';
+
+export const mapCatCPCData = (result: ResultUpload): DataField[] => {
+  const testResult = result.testResult as TestResultCatCPCSchema;
+  const t: TestData = result.testResult.testData as TestData;
+  const testDateTime = moment(testResult.journalData.testSlotAttributes.start, 'YYYY-MM-DDTHH:mm:ss');
+  const candidateDOB = moment(formatDateOfBirth(result)).format('YYMMDD');
+
+  const m: DataField[] = [
+    field('ACTIVITY_CODE', Number(testResult.activityCode)),
+    // unused - ADI_PRN
+    field('APP_REF_NO', formatApplicationReference(testResult.journalData.applicationReference)),
+    field('C', optionalBoolean(testResult, 'changeMarker')),
+    field('CANDIDATE_SURNAME', mandatory(testResult, 'journalData.candidate.candidateName.lastName')),
+    field('CAT_TYPE', formatCPCTestCategory(testResult)),
+    // unused - DATA_VALIDATION_FLAGS
+    field('DATE_OF_TEST', testDateTime.format('YYMMDD')),
+    field('DEBRIEF_GIVEN', testResult.activityCode === '51' ? 0 : 1),
```

Comment:
is there an enum value we can use instead of '51', or at least a constant with a name for clarity

Reply:
created const. No enum available here